backend/config/custom-environment-variables.json

```diff
@@ -169,22 +150,6 @@
   "slackAlerting": {
     "url": "CROWD_SLACK_ALERTING_URL"
   },
-  "sampleData": {
-    "tenantId": "CROWD_SAMPLE_DATA_TENANT_ID"
-  },
-  "unleash": {
```

Comment:
We still have unleash deployed on production - AFAIK we will still need it for feature flags for testing.

Reply:
Got green signal from Joana for removing it, since LF uses a different vendor. If we need feature flags in the future, we can probably build them from scratch ✌🏻